core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -240,19 +253,19 @@ func getExecuteMethodConfig(fromAddress string, routerProgramAddress string, com
 				},
 				IsSigner:   false,
 				IsWritable: false,
+				LookupOpts: chainwriter.LookupOpts{Optional: true},
 			},
 			chainwriter.PDALookups{
-				Name: "PerChainTokenConfig",
-				PublicKey: chainwriter.AccountConstant{
-					Address: routerProgramAddress,
-				},
+				Name:      "PerChainTokenConfig",
+				PublicKey: offrampAddressConstant(offrampProgramAddress),
 				Seeds: []chainwriter.Seed{
 					{Static: []byte("ccip_tokenpool_billing")},
 					{Dynamic: chainwriter.AccountLookup{Location: destTokenAddress}},
 					{Dynamic: chainwriter.AccountLookup{Location: destChainSelectorPath}},
 				},
 				IsSigner:   false,
 				IsWritable: false,
+				LookupOpts: chainwriter.LookupOpts{Optional: true},
```

Comment:
Which PDA does is this meant for? I don't think this one exists in the offramp. Also, it seems to overlap a bit with the next PDA, the one commented on in https://github.com/smartcontractkit/chainlink/pull/16214/files/8da7434561bf52781768c4d9700a9c7c3ad62fbc#r1949992852